kubernetes.yaml

```diff
@@ -87,6 +87,11 @@ data:
     launcher     = scrapyd_k8s.launcher.K8s
     
     namespace    = default
+    
+    max_proc     = 2
+    reconnection_attempts = 5
+    backoff_time = 5
+    backoff_coefficient = 2
```

Comment:
Can we have sensible defaults, and not have to think about this yet?
Then we can add a more detailed configuration documentation to document this. I think ideally, no one would need to touch these (unless you have very different network requirements) - I'd like the default to work for most cases.

Reply:
I think a section in the README would make sense here. Though might it be time to create a new file explaining the configuration options in more detail? That could give some more freedom to explain them in more detail.

Reply:
There are defaults in the code, but they need to be "tested" to see if it's enough for our prod cluster, I suspect that it might be possible that we need to increase the number of attempts, but I tried to make it resilient in a sense that every time the connection was successfully re-established, the number of attempts sets to provided default number, so we really catch the cases when we have several connection breaks in a row.

Reply:
Super, thanks! Pending:
- [ ] Remove the variables from the sample config files (defaults are meant to be ok, and are documented)
- [ ] Move config info from README.md to CONFIG.md
- [ ] Make sure CONFIG.md is linked to from README.md at a sensible place.

Reply:
Thank you for the feedback, done!